!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  Bf: 10.8529
  Br: 10.1507
  Cd: 0.8727
  Cf: -1.1705
  Cr: -1.1705
  Df: 2785.4
  Dr: 3152.3
  Vy_max: 5.0
  YawRate_max: 80.0
  dMtv: 1.0
  dRd: 0.2
  diff_delta: 1000.0
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      Bf: 10.8529
      Br: 10.1507
      Cd: 0.8727
      Cf: -1.1705
      Cr: -1.1705
      Df: 2785.4
      Dr: 3152.3
      Vy_max: 5.0
      YawRate_max: 80.0
      dMtv: 1.0
      dRd: 0.2
      diff_delta: 1000.0
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      latency: 5
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      q_mu: 0.0
      q_n: 5.0
      q_nN: 3.0
      q_s: 100.0
      q_sN: 100.0
      q_slack_track: 100.0
      q_slip: 2.0
      state: true
      type: ''
      u_r: 0.45
    state: []
  latency: 5
  q_mu: 0.0
  q_n: 5.0
  q_nN: 3.0
  q_s: 100.0
  q_sN: 100.0
  q_slack_track: 100.0
  q_slip: 2.0
  u_r: 0.45
state: []
